ComponentKit/Core/CKComponentViewConfiguration.mm

```diff
@@ -20,7 +20,11 @@
 
 CKComponentViewClass::CKComponentViewClass(Class viewClass) noexcept :
 identifier(class_getName(viewClass)),
-factory(^{return [[viewClass alloc] init];}) {}
+factory(^{ return [[viewClass alloc] init]; }) {
+  #if DEBUG
```

Comment:
I don't think it would hurt. Looks like the `DEBUG` check @zats mentions is from the initial commit. @adamjernst any idea if we need to keep that `DEBUG` check or if we can remove it too?